utilities/sanity.py

```
@@ -37,7 +37,7 @@ def storage_sanity_check(cluster_storage_classes_names: List[str]) -> bool:
         True if all expected storage classes from configuration exist on the cluster,
         False otherwise.
     """
-    config_sc = list([[*csc][0] for csc in py_config["storage_class_matrix"]])
+    config_sc = [[*csc][0] for csc in py_config["storage_class_matrix"]]
```

Ruff (0.15.12)

[warning] Prefer `next(iter(csc))` over single element slice

Replace with `next(iter(csc))`

(RUF015)